### PR TITLE
Url enhancement process interview

### PR DESCRIPTION
**Pull request by abessette**

### CODE REVIEW

File: interview/models.py [hunk 1]
```diff
@@ -66,6 +66,10 @@ class Candidate(models.Model):
     def __str__(self):
         return ("{name}").format(name=self.name)
 
+    @property
+    def name_slug(self):
+        return self.name.replace(" ", "_")
```
Comment (bport) [inline interview/models.py:71]: Use Django slugify function

File: interview/templates/interview/interview_minute.html [hunk 2]
```diff
@@ -7,10 +7,10 @@
 {% block content %}
 <div class="container">
     <div class="lead">
-        {% trans "Interview minute" %} <a href="{% url 'process-details' process_id=process.id %}"> {% trans "for process" %}</a> {% trans "concerning" %} <a href="{% url 'candidate' process.pk %}"> {{ process.candidate.name }} </a> {% trans "for subsidiary" %} {{ process.subsidiary.name }}
+        {% trans "Interview minute" %} <a href="{{ process.get_absolute_url }}"> {% trans "for process" %}</a> {% trans "concerning" %} <a href="{% url 'candidate' process.pk %}"> {{ process.candidate.name }} </a> {% trans "for subsidiary" %} {{ process.subsidiary.name }}
     </div>
     <h3>{% trans "Interviewer" %}</h3>
-    <ul>
+    <ul></ul>
```
Comment (bport) [inline interview/templates/interview/interview_minute.html:13]: I guess we don't want to close ul here

**Comment by abessette** [comment]
# Todo 

- [ ] Masser l'historique
- [ ] Ajouter l'évolution au changeLog
- [ ] Passer la mise à jour (orienté sécurité de Django)

**Comment by tuxella** [comment]
Could you use `_` between id, names, trigrams and ranks instead of `-` ?

Eg. `http://127.0.0.1:8000/interview/720_john-doe_abc_1/minute/` instead of `http://127.0.0.1:8000/interview/720-john-doe-abc-1/minute/`

**Comment by abessette** [comment]
> Could you use `_` between id, names, trigrams and ranks instead of `-` ?
> 
> Eg. `http://127.0.0.1:8000/interview/720_john-doe_abc_1/minute/` instead of `http://127.0.0.1:8000/interview/720-john-doe-abc-1/minute/`

Done !

File: interview/models.py (hunk 1 above)
Comment (ArnaudChirat) [reply interview/models.py]: Not possible, slugify use the - and not the _

**Pull request merged**